packages/pigeon/lib/pigeon_lib.dart

```diff
@@ -456,14 +456,13 @@ class _FindInitializer extends dart_ast_visitor.RecursiveAstVisitor<Object?> {
 }
 
 class _RootBuilder extends dart_ast_visitor.RecursiveAstVisitor<Object?> {
-  _RootBuilder(this.source, this.ignoresInvalidImports);
+  _RootBuilder(this.source);
```

Comment:
Are the changes to this file intended to be part of this PR? The description just mentions test cleanup.

Reply:
Yep, intended.  In a previous PR you mentioned that it was weird to have all the tests use this argument when in real usage we don't use it.  https://github.com/flutter/packages/pull/385#discussion_r658219560